common/shlibs

```diff
@@ -2751,10 +2751,10 @@ libTKXmlXCAF.so.11 oce-0.18_1
 libTKernel.so.11 oce-0.18_1
 libKF5CoreAddons.so.5 kcoreaddons-5.26.0_1
 librpmatch.so.0 musl-rpmatch-1.0_1
-libply.so.4 plymouth-0.9.2_1
-libply-splash-core.so.4 plymouth-0.9.2_1
-libply-splash-graphics.so.4 plymouth-0.9.2_1
-libply-boot-client.so.4 plymouth-0.9.2_1
+libply.so.5 plymouth-0.9.5_1
+libply-splash-core.so.5 plymouth-0.9.5_1
+libply-splash-graphics.so.5 plymouth-0.9.5_1
+libply-boot-client.so.5 plymouth-0.9.5_1
```

Comment:
Are these entries necessary at all? We could just clean them up.

Reply:
There shouldn't be any piece of software linking against `libply`, I'm fine removing those entries.